cargo-flash/README.md

```diff
@@ -1,8 +1,8 @@
 # cargo-flash
 
-This crate provides an utility to flash ARM chips.
+This crate provides an utility to download ELF binaries onto ARM chips.
```

Comment:
I actually prefer flash rather than download. Download could also mean we're just downloading it to RAM and executing it there.

Reply:
I totally see this :) Implemented.